src/dnx/dnx.vcxproj

```diff
@@ -15,6 +15,12 @@
     <ProjectName Condition="'$(BuildForOneCore)' == 'True'">dnx.onecore</ProjectName>
     <ConfigurationType>DynamicLibrary</ConfigurationType>
     <Defines Condition="$(RuntimeType) == 'CORECLR_WIN'">CORECLR_WIN</Defines>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Platform)' == 'ARM'">
+    <ProjectName>dnx</ProjectName>
+    <ConfigurationType>Application</ConfigurationType>
+  </PropertyGroup>
+  <PropertyGroup>
     <OutDir>bin\$(Platform)\$(Configuration)\$(TargetFramework)\</OutDir>
```

Comment:
"Globals" is just a label - it does not really matter. I will add them just for completeness.